File: src/Services/ActivityService/Models/Expense.cs
using System;

namespace ActivityService.Models
{
    public class Expense
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Category { get; set; } = "ostalo"; // prevoz, smještaj, hrana, ulaznice, kupovina, ostalo
        public decimal Amount { get; set; }
        public DateTime Date { get; set; }
        public string Description { get; set; } = string.Empty;
        public Guid TravelPlanId { get; set; }
    }
}

File: src/Frontend/src/models/Expense.js
export class Expense {
  constructor(data = {}) {
    this.id = data.id || '';
    this.name = data.name || '';
    this.category = data.category || 'ostalo'; // prevoz, smještaj, hrana, ulaznice, kupovina, ostalo
    this.amount = data.amount || 0;
    this.date = data.date ? new Date(data.date) : null;
    this.description = data.description || '';
    this.travelPlanId = data.travelPlanId || '';
  }
}
